examples/pipelines/ccfraud/submit.py

```diff
@@ -252,6 +259,28 @@ def fl_ccfraud_basic():
 
         # for each silo, run a distinct training with its own inputs and outputs
         for silo_index, silo_config in enumerate(YAML_CONFIG.federated_learning.silos):
+            # Determine number of processes to deploy on a given compute cluster node
+            ws_compute = ML_CLIENT.compute.get(silo_config.compute)
```

Comment:
happy to hear we can obtain this number here, could we put this under a distinct method? (something earlier in the script).

The goal is to enhance readability.